.github/policies/labelManagement.issueOpened.yml

```diff
@@ -93,5 +93,18 @@ configuration:
               label: Needs-Author-Feedback
           - addLabel:
               label: Manifest-Path-Error
+      - description: >-
+          When a pull request is opened, if the content contains "Hash Mismatch" or "Hash does not match" (case insensitive)
+          * Add the Update-Request label
+        if:
+          - payloadType: Issues
+          - isAction:
+              action: Opened
+          - bodyContains:
+              pattern: (?i)hash[\s-](mismatch|does\snot\smatch)
+              isRegex: True
+        then:
+          - addLabel:
+              label: Update-Request
```

Comment:
How about adding Error-Hash-Mismatch label too? Also this got me thinking, since hash mismatches are so common, maybe it's better we have another category for it in the issue templates that automatically applies that label. That'll help filtering out the "trivial" hash mismatch cases and the other non-obvious package issues..

Reply:
I think that adding the label automatically would work, since the Error-Hash-Mismatch trigger looks like it's only set to respond to PR's.

I don't think that having a separate issue template for it would really help at all, since many inexperienced users may simply classify it as a bug or a package issue. Also, the specific issue type for hash mismatches in the bug template would hit this trigger and apply the labels anyways